### PR TITLE
add guide button to launch IME settings activity

### PR DESCRIPTION
**Pull request by oldherl**
add a button and some text to guide the user to enable fcitx

### CODE REVIEW

File: app/src/main/java/me/rocka/fcitx5test/MainActivity.kt [hunk 1]
```diff
@@ -32,6 +34,11 @@ class MainActivity : AppCompatActivity() {
         binding = ActivityMainBinding.inflate(layoutInflater)
         val rootView = binding.root
         setContentView(rootView)
+        val openImeSettingsButton = findViewById<Button>(R.id.open_ime_settings)
+        openImeSettingsButton.setOnClickListener{
+            val intent = Intent(android.provider.Settings.ACTION_INPUT_METHOD_SETTINGS)
+            startActivity(intent)
+        }
```
Comment (rocka) [inline app/src/main/java/me/rocka/fcitx5test/MainActivity.kt:41]:
```suggestion
        findViewById<Button>(R.id.open_ime_settings).also {
            it.setOnClickListener {
                startActivity(Intent(android.provider.Settings.ACTION_INPUT_METHOD_SETTINGS))
            }
        }
```

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -7,14 +7,34 @@
     tools:context=".MainActivity">
 
     <TextView
-        android:id="@+id/input"
+        android:id="@+id/open_ime_settings_guide"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        app:layout_constraintBottom_toTopOf="@id/candidate"
+        android:text="@string/open_ime_settings_guide"
+        app:layout_constraintBottom_toTopOf="@id/commit"
```
Comment (rocka) [inline app/src/main/res/layout/activity_main.xml:14]:
```suggestion
        app:layout_constraintBottom_toTopOf="@id/open_ime_settings"
        app:layout_constraintVertical_chainStyle="packed"
```

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -7,14 +7,34 @@
...
+        app:layout_constraintBottom_toTopOf="@id/input"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/open_ime_settings_guide" />
+
+    <TextView
+        android:id="@+id/input"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintBottom_toTopOf="@id/commit"
```
Comment (rocka) [inline app/src/main/res/layout/activity_main.xml:33]:
```suggestion
        app:layout_constraintBottom_toTopOf="@id/candidate"
```

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -7,14 +7,34 @@
...
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/open_ime_settings" />
+
     <TextView
         android:id="@+id/candidate"
         android:layout_width="wrap_content"
```
Comment (rocka) [inline app/src/main/res/layout/activity_main.xml:40]:
```suggestion
        android:layout_width="match_parent"
```

**Review by rocka — COMMENTED** [review]
Guide text and button would overlap with test input content, just align them with ConstraintLayout.

**Review by rocka — CHANGES_REQUESTED** [review]
👀

File: app/src/main/java/me/rocka/fcitx5test/MainActivity.kt (hunk 1 above)
Comment (oldherl) [reply app/src/main/java/me/rocka/fcitx5test/MainActivity.kt:41]: kotlin :(

**Review by rocka — APPROVED** [review]
LGTM

**Pull request merged**